spec/controllers/idv/in_person/state_id_controller_spec.rb

```diff
@@ -228,7 +227,8 @@
         expect(pii_from_user[:dob]).to eq formatted_dob
         expect(pii_from_user[:identity_doc_zipcode]).to eq identity_doc_zipcode
         expect(pii_from_user[:identity_doc_address_state]).to eq identity_doc_address_state
-        expect(pii_from_user[:state_id_number]).to eq state_id_number
+        # param from form as id_number but is renamed to state_id_number on update
+        expect(pii_from_user[:state_id_number]).to eq id_number
```

Comment:
Instead of adding a new test to check that the params get translated back upon submit/"update" in the controller, I realized updating this test already verifies that happens. Just added this comment to illustrate that it is being checked here already.